Orange/data/variable.py

```diff
@@ -243,12 +250,31 @@ def __init__(self, name="", compute_value=None):
         self.unknown_str = MISSING_VALUES
         self.source_variable = None
         self.attributes = {}
+        self.master = self
         if name and compute_value is None:
             if isinstance(self._all_vars, collections.defaultdict):
                 self._all_vars[name].append(self)
             else:
                 self._all_vars[name] = self
 
+    def make_proxy(self):
+        """
+        Copy the variable and set the master to `self.master` or to `self`.
+
+        :return: copy of self
+        :rtype: Variable
+        """
+        var = copy.copy(self)
```

Comment:
`Variable.copy` is a misnomer. It should be named something like `derive`. I'm not sure I want to use it here, `copy.copy` looks safer since I want to have exactly the same thing. @astaric ?

I actually don't like what `Variable.copy` does. For instance, it does not copy the `colors` attribute, which the widget that I added is assigning to variables. I'm not even sure it does what it is intended for: `ContinuousVariable.copy` returns instances of `ContinuousVariable` even when `compute_value` is a discretizer.